### PR TITLE
Ts rewrite

### PR DESCRIPTION
**Pull request by x183**
Rewrite of lunch.chalmers.it in typescript, may have forgotten some functionality though

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=8080
```
Comment (Supergamer1337) [inline .env]: WHAT IS YOU .env DOING HERE?!?!?!?!?!?!?!?!?!?!?!?!?!?!?!?!??!?!?!?!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+old/
```
Comment (Supergamer1337) [inline .gitignore]: What is old/, why does this need to be here?

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Object.defineProperty(exports, "__esModule", { value: true });
+const express_1 = __importDefault(require("express"));
+const dotenv_1 = require("dotenv");
+const fetchRouter_1 = __importDefault(require("./routers/fetchRouter"));
+(0, dotenv_1.config)();
+const app = (0, express_1.default)();
+app.use("/", fetchRouter_1.default);
+app.listen(process.env.PORT, () => {
+    console.log(`Server is running on port ${process.env.PORT}`);
+});
```
Comment (Supergamer1337) [inline src/index.js]: What is your compile JS doing here?

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
Well I won't approve this unless you fix the .gitignore among other things (that is probably it though).

**Review by Supergamer1337 — CHANGES_REQUESTED** [review]
It seems it prints to the console *everytime* someone makes a request, with what that response would be. It would be better to print simply when getting new data, or if getting new data failed.

**Review by Supergamer1337 — APPROVED** [review]
Finally I can approve of this

File: README.md [hunk 4]
```diff
@@ -1,17 +1,9 @@
-# chalmersit-lunch fetcher
+# chalmersit-lunch
```
Comment (rhedinh) [inline README.md:1]: Add how-to for building project maybe?

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,1187 @@
...
+      "version": "1.12.2",
+      "resolved": "https://registry.npmjs.org/object-inspect/-/object-inspect-1.12.2.tgz",
+      "integrity": "sha512-z+cPxW0QGUp0mcqcsgQyLVRDoXFQbXOwBaqyF7VIgI4TWNQsDHrBpUQslRmIfAoYWdYzs6UlKJtB2XJpTaNSpQ=="
+    },
+    "on-finished": {
+      "version": "2.4.1",
+      "resolved": "https://registry.npmjs.org/on-finished/-/on-finished-2.4.1.tgz",
+      "integrity": "sha512-oVlzkg3ENAhCk2zdv7IJwd/QUD4z2RxRwpkcGY8psCVcCYZNq4wYnVWALHM+brtuJjePWiYF/ClmuDr8Ch5+kg==",
+      "requires": {
+        "ee-first": "1.1.1"
```
Comment (hampusrs) [inline package-lock.json]: Remove package-lock

File: README.md [hunk 6]
```diff
@@ -1,17 +1,9 @@
...
 
-
-## Instructions for Docker
-
-```bash
-  docker build -t cthit/lunch .
-  docker run --rm -p 3000:3000 cthit/lunch
-```
\ No newline at end of file
+- [@x183](https://github.com/x183/) Oscar 'saxen' Palm
```
Comment (hampusrs) [inline README.md]: Add how-to for building project

**Review by hampusrs — CHANGES_REQUESTED** [review]
This would be nice

**Comment by hampusrs** [comment]
Also please install git in the repo :')

File: README.md (hunk 6 above)
Comment (x183) [reply README.md]: Is fixed

**Comment by hampusrs** [comment]
LGTM

**Review by hampusrs — APPROVED** [review]

**Pull request merged**